.etckeeper

```diff
@@ -0,0 +1,1386 @@
+maybe chmod 0644 'modules'
+maybe chmod 0755 'modules-load.d'
+maybe chmod 0644 'motd'
+maybe chmod 0644 'msmtprc'
+maybe chmod 0644 'Muttrc'
+maybe chmod 0755 'Muttrc.d'
+maybe chmod 0644 'Muttrc.d/charset.rc'
+maybe chmod 0644 'Muttrc.d/colors.rc'
+maybe chmod 0644 'Muttrc.d/compressed-folders.rc'
+maybe chmod 0644 'Muttrc.d/gpg.rc'
+maybe chmod 0644 'Muttrc.d/sidebar.rc'
+maybe chmod 0644 'Muttrc.d/smime.rc'
+maybe chmod 0755 'mysql'
+maybe chmod 0755 'mysql/conf.d'
+maybe chmod 0644 'mysql/conf.d/.keepme'
+maybe chmod 0644 'mysql/my.cnf'
+maybe chmod 0644 'nanorc'
+maybe chmod 0755 'nethack'
+maybe chmod 0644 'nethack/nethackrc.console'
+maybe chmod 0644 'nethack/nethackrc.tty'
+maybe chmod 0755 'network'
+maybe chmod 0755 'network/if-down.d'
+maybe chmod 0755 'network/if-down.d/postfix'
+maybe chmod 0755 'network/if-down.d/resolvconf'
+maybe chmod 0755 'network/if-down.d/upstart'
+maybe chmod 0755 'network/if-post-down.d'
+maybe chmod 0755 'network/if-pre-up.d'
+maybe chmod 0755 'network/if-up.d'
+maybe chmod 0755 'network/if-up.d/000resolvconf'
+maybe chmod 0755 'network/if-up.d/mountnfs'
+maybe chmod 0755 'network/if-up.d/nslcd'
+maybe chmod 0755 'network/if-up.d/openntpd'
+maybe chmod 0755 'network/if-up.d/openssh-server'
+maybe chmod 0755 'network/if-up.d/postfix'
+maybe chmod 0755 'network/if-up.d/upstart'
+maybe chmod 0755 'network/interfaces.d'
+maybe chmod 0644 'network/interfaces.example'
+maybe chmod 0755 'newt'
+maybe chmod 0644 'newt/palette.original'
+maybe chmod 0644 'nova-agent.env'
+maybe chmod 0644 'npmrc'
+maybe chmod 0644 'nscd.conf'
+maybe chmod 0644 'nslcd.conf'
+maybe chmod 0644 'nsswitch.conf'
+maybe chmod 0644 'oidentd.conf'
+maybe chmod 0644 'oidentd_masq.conf'
+maybe chmod 0755 'openal'
+maybe chmod 0644 'openal/alsoft.conf'
+maybe chmod 0755 'openntpd'
+maybe chmod 0644 'openntpd/ntpd.conf'
+maybe chmod 0755 'opt'
+maybe chmod 0600 'packages.txt'
+maybe chmod 0644 'pam.conf'
+maybe chmod 0755 'pam.d'
+maybe chmod 0644 'pam.d/atd'
+maybe chmod 0644 'pam.d/chfn'
+maybe chmod 0644 'pam.d/chpasswd'
+maybe chmod 0644 'pam.d/chsh'
+maybe chmod 0644 'pam.d/common-account'
+maybe chmod 0644 'pam.d/common-auth'
+maybe chmod 0644 'pam.d/common-password'
+maybe chmod 0644 'pam.d/common-session'
+maybe chmod 0644 'pam.d/common-session-noninteractive'
+maybe chmod 0644 'pam.d/cron'
+maybe chmod 0644 'pam.d/login'
+maybe chmod 0644 'pam.d/newusers'
+maybe chmod 0644 'pam.d/other'
+maybe chmod 0644 'pam.d/passwd'
+maybe chmod 0644 'pam.d/polkit-1'
+maybe chmod 0644 'pam.d/runuser'
+maybe chmod 0644 'pam.d/runuser-l'
+maybe chmod 0644 'pam.d/sshd'
+maybe chmod 0644 'pam.d/su'
+maybe chmod 0644 'pam.d/sudo'
+maybe chmod 0644 'pam.d/systemd-user'
+maybe chmod 0644 'papersize'
+maybe chmod 0755 'parallel'
+maybe chmod 0644 'passwd'
+maybe chmod 0755 'perl'
+maybe chmod 0755 'perl/CPAN'
+maybe chmod 0755 'perl/Net'
+maybe chmod 0644 'perl/Net/libnet.cfg'
+maybe chmod 0644 'perl/sitecustomize.pl'
+maybe chmod 0755 'php5'
```

Comment:
I think `.etckeeper` should just be outright deleted, if we are rewriting history anyhow.